src/main/java/tests/ShopServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package tests;
+
+import core.basesyntax.*;
```

Comment:
It's better to import only the necessary classes rather than using a wildcard import. This can help to understand which classes are actually being used in the test.